modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -719,6 +719,17 @@ public static real_t Sin(real_t s)
             return (real_t)Math.Sin(s);
         }
 
+        /// <summary>
+        /// Returns the sine and cosine of angle <paramref name="s"/> in radians.
+        /// </summary>
+        /// <param name="s">The angle in radians.</param>
+        /// <returns>The sine and cosine of that angle.</returns>
+        public static (real_t Sin, real_t Cos) SinCos(real_t s)
```

Comment:
This should go in `MathfEx.cs`, the file for math APIs present in C# but not in core.